src/store/selectors.js

```diff
@@ -0,0 +1,4 @@
+export const selectCourses = (state) => state.courses;
+export const selectAuthors = (state) => state.authors.authors;
+export const selectCourseAuthors = (state) => state.authors.courseAuthors;
```

Comment:
this shouldn't be here, please see the comments about store structure

Reply:
Changed